pyproject.toml

```diff
@@ -17,6 +16,30 @@ show_error_codes = true
 module = ["tests.*"]
 disallow_untyped_defs = false
 
+[[tool.mypy.overrides]]
```

Comment:
This is different. This allows us to not type the tests. A lot of the tests don't currently have types so mypy has a lot of errors on the tests directory for that reason. I don't think types on the tests really provide any benefits, so this allows leaving them off rather than taking the time to go back and type everything.

Reply:
I guess to rephrase, I'm not an expert in `toml` files, so I really meant, was it okay to have multiple `[[tool.mypy.overrides]]` headings, that they were both applied independently, and the second doesn't override the first?  It must be fine since we're not getting the mypy errors on the tests.

Reply:
ah, ok this is how you specify different rules in the toml file for mypy. So each independent rule is its own `[[tool.mypy.overrides]]`.

Reply:
got it, thanks for explaining!